modules/security-context-constraints-about.adoc

```diff
@@ -143,10 +143,7 @@ The `restricted` SCC:
 * Allows pods to use any FSGroup
 * Allows pods to use any supplemental group
 
-[NOTE]
-====
-The restricted SCC is the most restrictive of the SCCs that ship by default with the system. However, you can create a custom SCC that is even more restrictive. For example, you can create an SCC that restricts `readOnlyRootFS` to `true` and `allowPrivilegeEscalation` to `false`.
-====
+This SCC is used by default for authenticated users for clusters that were upgraded from {product-title} 4.10 or earlier. [^1^]
```

Comment:
I would avoid the "used by default" here. "restricted-v2" is used by default because it has higher priority.

Maybe consider something like:
"In clusters that were upgraded from {product-title} 4.10 or earlier, this SCC is available for use by any authenticated user."